<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-DKK5M0P" failures="2" tests="4" name="Test" time="30.851" errors="0" timestamp="2025-02-07T22:05:35 IST">
  <testcase classname="tests.ErrorValidation" name="loginNegativeCheck" time="0.858">
    <skipped/>
  </testcase> <!-- loginNegativeCheck -->
  <testcase classname="tests.ErrorValidation" name="loginNegativeCheck" time="0.478">
    <failure type="java.lang.AssertionError" message="expected [Incorrect email or password.] but found [Incorrect email password.]">
      <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at tests.ErrorValidation.loginNegativeCheck(ErrorValidation.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginNegativeCheck -->
  <testcase classname="tests.ErrorValidation" name="productNegativeCheck" time="13.95">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at tests.ErrorValidation.productNegativeCheck(ErrorValidation.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- productNegativeCheck -->
  <testcase classname="tests.ErrorValidation" name="validateOrders" time="0.0">
    <skipped/>
  </testcase> <!-- validateOrders -->
</testsuite> <!-- Test -->
